### DISCUSSION TITLE
How to get the Output File Name into the <title> field

### DISCUSSION THREAD

**Discussion by gstozier** [enzo1982/freac#463] [Help & Support]
I've found that several apps use the <title> field to id the track instead of the external file name.
How can I get the <title> field filled in from the <filename>?
Thanks,
Steve
![fesamp](https://user-images.githubusercontent.com/129112512/228050666-d7e55974-f167-4664-8ba4-1ff624ea41b1.jpg)

**Comment by mrfragger** [comment]
easiest is kid3-cli which is mac, linux, win
`for f in *.opus ; do kid3-cli -c "set title '$(printf '%n' "$f")'" "$f" ; done`

I combine it with python titlecase  Lord Of The Rings --> Lord of the Rings
`for f in *.opus ; do mv -i -- "$f" "$(printf '%s\n' "$f" | titlecase)" ; kid3-cli -c "set title '$(printf '%n' "$f")'" "$f" ; done`

If you want a GUI to do it
Title to Tag   use puddletag
pip install puddletag
set the syntax to %title%
select all files Ctrl-A
 and then click the T next to %title% which is "Convert filename to tag using pattern"
then be sure to SAVE.
